intel_extension_for_transformers/llm/runtime/graph/models/llama/llama.cpp

```diff
@@ -149,18 +151,25 @@ static bool llama_model_eval_internal(model_context& lctx, const model_token* to
           ne_reshape_3d(ctx0, ne_view_1d(ctx0, QKVcur, N * n_embd, 1 * N * n_embd * ne_element_size(QKVcur)),
                         n_embd / n_head, n_head, N),
           n_past, n_rot, 0, 0);
-      Vcur = ne_transpose(
-          ctx0, ne_reshape_2d(ctx0, ne_view_1d(ctx0, QKVcur, N * n_embd, 2 * N * n_embd * ne_element_size(QKVcur)),
-                              n_embd, N));
+      if (n_head == n_head_kv) {
+        Vcur = ne_transpose(
+            ctx0, ne_reshape_2d(ctx0, ne_view_1d(ctx0, QKVcur, N * n_embd, 2 * N * n_embd * ne_element_size(QKVcur)),
+                                n_embd, N));
+      } else {
+        Vcur = ne_transpose(
+            ctx0, ne_reshape_2d(ctx0, ne_view_1d(ctx0, QKVcur, N * n_embd, 2 * N * n_embd * ne_element_size(QKVcur)),
```

Comment:
`offset` in `ne_view_1d` = `(N*n_embd + N*n_embd_gqa) * ne_element_size(QKVcur)`?